docs/user-docs/annotation.md

```diff
@@ -1290,6 +1292,7 @@ The following attributes can be used to manipulate the presentation settings of
       - `ulist` for unordered bullet list.
       - `csv` for comma-seperated values.
       - `raw` for space-seperated values.
+  - `"selector_ux_mode"`: The display mode for the recordedit input field when this column directive is a foreign key relationship, either from the foreign key name or a source path with a single outb ound table. Supported values are `"popup"` and `"dropdown"`, with `"popup"` being the default. Currently only supported in `entry` contexts.
```

Comment:
I don't think you need to say all of the "is a foreign key relationship, either from the foreign key name or a source path with a single out bound table". It makes it harder to read. 

I think you should use the exact same explanation as the one you wrote for `foreign-key` annotation. I think that would imply that this only applies only if the column directive is a single-hop outbound foreign key path without needing to explicitly mention it.